src/maxKey.js

```diff
@@ -8,13 +8,25 @@
  * @param {object} array - A non-empty array of numbers
  * @returns {number} The key with the largest value
  */
+
+
 function maxKey(obj) {
-  // This is your job. :)
+  var dict = {};
+var arr = [];
+for (var key in dict) {
+    if (dict.hasOwnProperty(key)) {
+        arr.push( [ key, dict[key] ] );
+    }
+}
+  return Math.max.apply(null,Object.keys(obj));
```

Comment:
This will give you the largest element in the array you pass in. You're passing in `Object.keys(obj)`, which is the array of keys in `obj`.

Therefore, this will return the alphabetically largest key, not the key whose value is the largest. That is, it'll give you the key that comes last alphabetically when compared to the other keys in `obj`, regardless of the associated value.